subprojects/gradle/slack/src/main/kotlin/com/avito/slack/model/FoundMessage.kt

```diff
@@ -2,9 +2,9 @@ package com.avito.slack.model
 
 data class FoundMessage(
     val text: String,
-    val botId: String,
+    val botId: String?,
     val timestamp: String,
-    val author: String,
+    val author: String?,
```

Comment:
This could be service messages, and user sees integration hook "name" as username, but in fact message author fetched as null